docs/what_combine_does/model_and_likelihood.md

```diff
@@ -0,0 +1,396 @@
+
+[Combining full models](../../part2/settinguptheanalysis/#combination-of-multiple-datacards) is possible by combining their channels, assuming that the channels are mutually independent.
+
+/// details | **A Simple Example** 
+
+Consider performing an analysis searching for a Higgs boson by looking for events where the Higgs decays into two photons.
+
+The event count data may be binned histograms of the number of events with two photons with different bins of invariant mass of the photons.
+The expected counts would include signal contributions from processes where a Higgs boson is produced, as well as background contributions from processes where two photons are produced through other mechanisms, like radiation off a quark. 
+The expected counts may also depend on parameters such as the energy resolution of the measured photons and the total luminosity of collisions being considered in the dataset, these can be parameterized in the model as auxiliary information.
+
+The analysis itself might be split into multiple channels, targetting different Higgs production modes with different event selection criteria.
+Furthermore, the analysis may eventually be combined with other analyses, such as a measurement targetting Higgs production where the Higgs boson decays into four leptons, rather than two photons.
+
+Combine provides the functionality for building the statistical models and combining all the channels or analyses together into one common analysis.
+
+
+///
+
+### Sets of Observation Models
+
+We are typically not interested in a single model, but in a set of models, parameterized by a set of real numbers representing possible versions of the model. 
+
+Model parameters include the parameters of interest ( $\vec{\mu}$, those being measured such as a cross section) as well as nuisance parameters ($\vec{\nu}$), which may not be of interest but still affect the model expectation.
+
+Combine provides tools and interfaces for defining the model as pre-defined or user-defined functions of the input parameters.
+In practice, however, there are a number of most commonly used functional forms which define how the expected events depend on the model parameters.
+These are discussed in detail in the context of the full likelihood below.
+
+## The Likelihood 
+
+For any given model, $\mathcal{M}(\vec{\Phi})$, [the likelihood](https://pdg.lbl.gov/2022/web/viewer.html?file=../reviews/rpp2022-rev-statistics.pdf#section.40.1) defines the probability of observing a given dataset. 
+It is numerically equal to the probability of observing the data, given the model. 
+
+$$ \mathcal{L}_\mathcal{M}(\vec{\Phi};\mathrm{data}) = p_{\mathcal{M}}(\mathrm{data};\vec{\Phi}) $$
+
+Note, however that the likelihood is a function of the model parameters, not the data, which is why we distinguish it from the probability itself. 
+
+The likelihood in combine takes the general form:
+
+$$ \mathcal{L} =  \mathcal{L}_{\textrm{data}} \cdot \mathcal{L}_{\textrm{constraint}} $$
```

Comment:
can we use "primary" and "auxiliary" as in the paper? instead of data and constraint ?